memphis/message.py

```diff
@@ -37,6 +38,34 @@ async def ack(self):
                 raise MemphisConnectError(str(e)) from e
             return
 
+    async def nack(self):
+        """
+        nack - not ack for a message, meaning that the message will be redelivered again to the same consumers group without waiting to its ack wait time.
+        """
+        await self.message.nak()
+
+    async def dead_letter(self, reason: str):
+        """
+        dead_letter - Sending the message to the dead-letter station (DLS). the broker won't resend the message again to the same consumers group and will place the message inside the dead-letter station (DLS) with the given reason.
+        The message will still be available to other consumer groups
+        """
+        try:
+            await self.message.term()
```

Comment:
what will happen in case the message has not term, like nats message.
Just a reminder we don't want an error to be thrown to the user